se/se_epub_lint.py

```diff
@@ -262,7 +262,6 @@
 "s-071", "Sectioning element with more than one heading element."
 "s-072", "Element with single [xhtml]<span>[/] child. [xhtml]<span>[/] should be removed and its attributes promoted to the parent element."
 "s-073", "Header element that requires [val]label[/] and [val]ordinal[/] semantic children."
-"s-074", "[xhtml]<hgroup>[/] element containing sequential [xhtml]<h#>[/] elements at the same heading level."
```

Comment:
Good to list removed rules as available for the next time we want to add one.

Reply:
I didn't know the tests had a core.css; I'll add that.

I used the structure Alex initially posted. These are epubs, not high-traffic web sites, so I can't imagine that makes the slightest difference at all in performance on an ereader, but I'll be glad to make the change if Alex prefers.

List them where? They're available if they're not being used. :)

Reply:
Given the state of CPUs on ereaders I’d say it’s more important here than on a website, but it’s a minimal different either way.

Have a look at this commit for an example of retiring a lint rule: https://github.com/standardebooks/tools/commit/98fa396d29f514ab8e0c7b92de3f4b4e2fc0aa37

Reply:
To be clear: I don't care which we use, and what you suggest is what I would have done had I written it from scratch. But I learned long ago that when Alex suggests CSS, that's the CSS to use. :)

That UNUSED block no longer exists. I'm sure there's a way in git to figure out when it went away, but I don't know what it is (without looking at every commit involving se_epub_lint, which I'm not going to do). Regardless, it's not there now. And, again, anything that is not in the used list is by definition unused, so I don't understand why it would be needed anyway.

Reply:
Add a new equivalent block to the bottom of the `s-` rules. The block is removed when those unused rules are all reused. It’s just there as a reference so that those numbers can be reused in future rule additions. Keeps the numbering in check, and everyone’s OCD happy.